### DISCUSSION TITLE
monday-sdk-js - Moving types to the monday-sdk-js library

### DISCUSSION THREAD

**Discussion by gregra81** [DefinitelyTyped/DefinitelyTyped#64287] [Issues with a @types package] [answered]
Hey @josh-, Greg from monday here. 

First of all - thank you for maintaining the types for monday-sdk-js, this is super helpful for many monday developers!
We're in the process of adding types directly within our repo - https://github.com/mondaycom/monday-sdk-js and:
1. Wanted to let you know and thank you again for maintaining them until now
2. Suggesting you that going forward, you can feel free to contribute to the SDK repo, if you'd like

Have a great day!

[a comment by an automated account is not shown]

**Comment by josh-** [comment, marked as answer]
Thanks @gregra81 - great news!

Once the types are published in the Monday SDK package I'll run the [steps here for removing a package from DT](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) here 👍
